derivation-tests/shared/src/test/scala/zio/prelude/derivation/tests/DeriveDebugSpec.scala

```diff
@@ -0,0 +1,59 @@
+package zio.prelude.derivation.tests
+
+import zio._
+import zio.prelude._
+import zio.prelude.derivation.DeriveDebug
+import zio.test._
+
+object DeriveDebugSpec extends ZIOSpecDefault {
+  sealed trait Entity
+  object Entity {
+    case class Person(name: String, address: Address) extends Entity
+    object Person {
+      implicit val debug: Debug[Person] = DeriveDebug.gen[Person]
```

Comment:
Would be good to have an annotation for this. Something like `@deriving(Debug)`.

Reply:
Could you elaborate more on this? Do you mean to have some annotations like `@jsonHint` so that we can fine tune the generated Debug instance?